presto-native-execution/src/test/java/com/facebook/presto/spark/TestPrestoSparkNativeSimpleQueries.java

```diff
@@ -119,4 +119,50 @@ public void testJsonFileBasedFunction()
         assertQuery("SELECT json.test_schema.eq(1, linenumber) FROM lineitem", "SELECT 1 = linenumber FROM lineitem");
         assertQuery("SELECT json.test_schema.sum(linenumber) FROM lineitem", "SELECT sum(linenumber) FROM lineitem");
     }
+
+    @Test
+    public void testAggregationCompanionFunction()
```

Comment:
These queries do not reflect the way companion functions are designed to be used. Would it be possible to create more realistic tests by (1) evaluating 'partial' aggregation and save results to a table; (2) evaluate 'final' aggregation on results of (1).